spec/ajax-datatables-rails/base_spec.rb

```diff
@@ -86,6 +86,26 @@
           expect(item[2]).to eq 'Name &quot;&gt;&lt;img src=x onerror=alert(&quot;first_name&quot;)&gt;'
           expect(item[3]).to eq 'Name &quot;&gt;&lt;img src=x onerror=alert(&quot;last_name&quot;)&gt;'
         end
+
+        context 'with no searchable columns' do
+          let(:datatable) do
+            ComplexDatatableArray.new(view).tap do |datatable|
+              datatable.instance_variable_set :@searchable_columns, []
+            end
+          end
+
+          context 'when search value is comma-separated string' do
+            let(:view) { double('view', params: sample_params.merge(search: { value: 'one, two' })) }
+
+            it 'should return an array of arrays' do
```

Comment:
This test fails without https://github.com/jbox-web/ajax-datatables-rails/pull/235/commits/45fde7e3780c2a40275d08ad4b01156cf33d2f17.